test/cljs/chronograph/test_utils.cljs

```diff
@@ -0,0 +1,13 @@
+(ns chronograph.test-utils
+  (:require [re-frame.core :as rf]
+            [bidi.bidi :as bidi]
+            [chronograph-web.events.routing :as routing-events]
+            [chronograph-web.routes :as routes]))
+
+(defn set-token [token]
+  (rf/dispatch [::routing-events/pushy-dispatch (bidi/match-route routes/routes token)]))
+
+(defn stub-routing []
```

Comment:
`stub-routing` does not seem to be used anywhere in this PR, at the time of this comment. Is it being used somewhere?

Reply:
Yes, it is being used.